front-end/src/Components/ProductCard.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styled from "styled-components";
+
+import { device } from "../utils/device";
+
+const ProductCardWrapperStyled = styled.section`
+  display: block;
+  position: relative;
+  border-style: solid;
+  border-width: 0 100vw 40vh 0;
+  border-color: transparent #97bf04 transparent transparent;
+`;
+
+const ProductCardStyled = styled.div`
```

Comment:
Can we move all styling stuff to css files ?

Reply:
i used styled-components library so far in this project, but we can decide for css modules if you want and I will refactor all files